### PR TITLE
Fix `cupy.random.bytes` not working

### LINKED ISSUE #4312: cupy.random.bytes not working
* Conditions (you can just paste the output of `python -c 'import cupy; cupy.show_config()'`)

```
OS                           : Linux-5.9.8-2-MANJARO-x86_64-with-arch-Manjaro-Linux
CuPy Version                 : 8.1.0
NumPy Version                : 1.19.4
SciPy Version                : 1.5.3
CUDA Root                    : /[hiden]/.conda/envs/rapids-0.16
CUDA Build Version           : 10010
CUDA Driver Version          : 11010
CUDA Runtime Version         : 10010
cuBLAS Version               : 10201
cuFFT Version                : 10101
cuRAND Version               : 10101
cuSOLVER Version             : (10, 2, 0)
cuSPARSE Version             : 10300
NVRTC Version                : (10, 1)
Thrust Version               : 100906
CUB Build Version            : 100800
cuDNN Build Version          : None
cuDNN Version                : None
NCCL Build Version           : 2708
NCCL Runtime Version         : 2708
cuTENSOR Version             : None
Device 0 Name                : GeForce GTX 1080 Ti
Device 0 Compute Capability  : 61
```

* Code to reproduce
```python
import cupy as cp
b = cp.random.bytes(10)
```

* Error messages, stack traces, or logs
```
---------------------------------------------------------------------------
AttributeError                            Traceback (most recent call last)
<ipython-input-121-104e7c75af44> in <module>
      1 import cupy as cp
----> 2 b = cp.random.bytes(10)

~/.conda/envs/rapids-0.16/lib/python3.7/site-packages/cupy/random/__init__.py in bytes(length)
      8                  <numpy.random.mtrand.RandomState.bytes>`
      9     """
---> 10     return _numpy.bytes(length)
     11 
     12 

~/.conda/envs/rapids-0.16/lib/python3.7/site-packages/numpy/__init__.py in __getattr__(attr)
    213             else:
    214                 raise AttributeError("module {!r} has no attribute "
--> 215                                      "{!r}".format(__name__, attr))
    216 
    217         def __dir__():

AttributeError: module 'numpy' has no attribute 'bytes'
```

### PR DESCRIPTION
**Pull request by kmaehashi**
This resolves the bug #4312

### CODE REVIEW

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Comment by lucasrodes** [comment]
Thanks, @kmaehashi

File: cupy/random/__init__.py [hunk 1]
```diff
@@ -4,10 +4,14 @@
 def bytes(length):
     """Returns random bytes.
 
+    .. note:: This function is just a wrapper for :meth:`numpy.random.bytes`.
```
Comment (toslunar) [inline cupy/random/__init__.py:7]: The link is broken on the CI (https://cupy--4318.org.readthedocs.build/en/4318/reference/generated/cupy.random.bytes.html).  While `numpy.random.mtrand.RandomState.bytes` is a method, `numpy.random.bytes` seems a function (`:func:`).

Comment (toslunar) [reply cupy/random/__init__.py:7]: `:obj:` works in the NumPy-CuPy comparison table.

**Comment by kmaehashi** [comment]
pfnCI, test this please.

**Review by toslunar — APPROVED** [review]

**Pull request merged**